packages/core/event.ts

```diff
@@ -105,6 +113,8 @@ export const validateCustomEventName = (
     "{Organiser}",
     "{Scheduler}",
     "{Location}",
+    "{Organiser first name}",
```

Comment:
as mentioned in https://github.com/calcom/cal.com/pull/14156#pullrequestreview-2012481181, we wanted `{Scheduler first name}` and `{Scheduler last name}`

But you can leave `{Organiser first name}`, that's fine

Reply:
The issue stated first name, which is why I did not include last name. I will make changes to it. 